lively.project/project.js

```diff
@@ -236,16 +270,16 @@ export class Project {
           await loadedProject.gitResource.changeRemoteURLToUseCurrentToken(currUserToken, repoOwner, name);
         }
       }
-      if (!lively.isInOfflineMode) {
+      if (!lively.isInOfflineMode && !lively.projectRepoPull) {
```

Comment:
When will this condition be true?

Reply:
Only if the prefetch did not happen. I did not want the projects to be dependent on bootstrap code in order to run on their own, so this is a safeguard for that.